test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -353,7 +369,18 @@ describe('Test gl plot interactions', function() {
                         .toBeCloseToArray([1.25, 1.25, 1.25], 4);
 
                     selectButton(modeBar, 'resetCameraLastSave3d').click();
+
                     setTimeout(function() {
+
+                        expect(relayoutCallback).toHaveBeenCalled(); // initiator: resetCameraLastSave3d
+                        expect(relayoutCallback).toHaveBeenCalledWith({
+                            scene: {
+                                eye: { x: 1.25, y: 1.25, z: 1.25 },
```

Comment:
This assertion is misleading. Adding

```js
console.log(getOrbitCamera(cameraData))
```

above this [line](https://github.com/monfera/plotly.js/blob/29-emit-relayout-events-on-3d-camera-reset/src/plots/gl3d/scene.js#L599) in `Scene.prototype.setCamera` logs

```js
[[0.1, 0.1, 1], [0, 0, 0], [0, 0, 1]]
```

for the first scene, and

```js
[[2.5, 2.5, 2.5], [0, 0, 0], [0, 0, 1]]
```

for the second scene.

I think the jasmine spy, holds on to the arguments of the first `relayoutCallback` call and isn't updated for subsequent calls.

Moreover, we should make it clear that `plotly_relayout` is triggered twice (one per each scene) when clicking on these buttons.